js/blocks.js

```diff
@@ -1655,7 +1655,7 @@ class Blocks {
                     }
 
                     /** Look for available connections. */
-                    if (this._testConnectionType(blkType, this.blockList[b].docks[i][2])) {
+                    if ((this._testConnectionType(blkType, this.blockList[b].docks[i][2])) || (this.blockList[b].name == "switch" && myBlock.name == "vspace") || (this.blockList[b].name == "vspace" && myBlock.name == "switch")) {
```

Comment:
I might be better to put logic into testConnectionType to allow a vspace to connect to a `caseout` type rather than adding a special case here.

Reply:
@walterbender, I was trying that at first. But the issue I was facing was that if I connected them by verifying the dock types then due to many blocks having similar dock types, all those other blocks were also able to connect to the switch block.

Reply:
Okay....I'll get right on it.